### PR TITLE
[WIP] Volodymyrk/83 remove global from styles

### PR DESCRIPTION
**Pull request by VKobeliatsky**
- extracted `:global` styles from `base.st.css` into separate stylesheets;
 - used those stylesheets in demos;
 - `<DatePicker />` uses input stylesheet for its `<input />` subcomponent;

### CODE REVIEW

**Comment by tyv** [comment]
@giladgrush take a look, is it ok for now?

File: src/style/default-theme/typography.st.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    line-height: 24px;
+}
+
+.label {
+    color: value(color_MainPlaceholder);
+    font-size: 16px;
+    line-height: 24px;
+}
+
+.img {
```
Comment (giladgrush) [inline src/style/default-theme/typography.st.css]: img in typography?

**Comment by giladgrush** [comment]
@VKobeliatsky note that they're working on adding theme support for stylable which would make it possible to create a single "theme" file in which you can import all the other st.css files. This will remove the need for all the imports under different names in the demo.

File: src/style/default-theme/typography.st.css (hunk 1 above)
Comment (VKobeliatsky) [inline src/style/default-theme/typography.st.css]: I imagine that anything that can appear in a newspaper goes to typography. Can you suggest a better place for `.img`?

**Comment by VKobeliatsky** [comment]
@giladgrush cool! I'll have to see an example first, though.

File: src/style/default-theme/typography.st.css (hunk 1 above)
Comment (giladgrush) [inline src/style/default-theme/typography.st.css]: Sure, talk to Avi, move it to the image component branch or if not needed there, you can remove it altogether.

**Pull request closed without merging**